DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/GeometryUtils.cs

```diff
@@ -0,0 +1,54 @@
+using ArcGIS.Core.CIM;
+
+namespace Speckle.Converters.ArcGIS3.Geometry;
+
+public class GeometryUtils : IGeometryUtils
+{
+  public bool ValidateMesh(SOG.Mesh mesh)
+  {
+    if (mesh.vertices.Count < 3)
+    {
+      return false;
+    }
+    else if (mesh.faces.Count < 4)
+    {
+      return false;
+    }
+    return true;
+  }
+
+  public int RGBToInt(CIMRGBColor color)
+  {
+    return (255 << 24) | ((int)Math.Round(color.R) << 16) | ((int)Math.Round(color.G) << 8) | (int)Math.Round(color.B);
+  }
+
+  public int CIMColorToInt(CIMColor color)
+  {
+    return (255 << 24)
+      | ((int)Math.Round(color.Values[0]) << 16)
+      | ((int)Math.Round(color.Values[1]) << 8)
+      | (int)Math.Round(color.Values[2]);
+  }
+
+  public bool IsClockwisePolygon(List<SOG.Point> points)
```

Comment:
This signature is kinda odd, a list of point's in speckle doesn't necessarily represent a Polygon. Is there no higher class in speckle that you can use? 

I'm assuming these points will become a polyline, in which case why not have the polyline as input instead? This could allow us to make this an extension method of that class.

I'm also not fully certain why we need to be doing this operations. Why do we care? :)

Reply:
Indeed, Polyline can be a better input 👌 The method is called in different scenarios:
1. Multipolygons are written as a list of ArcGIS Polylines, and the only way to understand what is what (outer vs. inner ring) is by orientation. 
2. When we have a simple outer-ring polygon, it will have a 1-faced Mesh created for the displayValue. And in this case, the order of the points is important to make the mesh facing upwards.